lib/util/constants.js

```diff
@@ -72,11 +72,6 @@ module.exports.DEFAULT_NQ_LEVEL_LOCAL = 1;
 module.exports.DEFAULT_NQ_LEVEL_REMOTE = 0;
 module.exports.MAX_NQ_LEVEL = 3;
 
-// eslint-disable-next-line no-warning-comments
-// TODO(mmalavalli): Once we decide to support Unified Plan on Chrome 72+,
-// we need to remove this constant and its references.
-module.exports.DEFAULT_CHROME_SDP_SEMANTICS = 'plan-b';
```

Comment:
@makarandp0 Please bring this back. Otherwise we will still be using unified plan on Chrome.

Reply:
Thanks for catching this.